src/dvclive/dvc.py

```diff
@@ -156,3 +157,29 @@ def get_random_exp_name(scm, baseline_rev):
         exp_ref = ExpRefInfo(baseline_sha=baseline_rev, name=name)
         if not scm.get_ref(str(exp_ref)):
             return name
+
+
+def get_dvc_stage_template(live):
+    stage = {
+        "cmd": "<python my_code_file.py my_args>",
```

Comment:
WDYT of just showing `python train.py` without `<>` syntax or anything.

Reply:
🤔 I worry it will confuse users (although anything we put here might); ideally I'd like to make clear that it's up to the user to fill in the `cmd`. Why do you prefer `python train.py` and do you have any other ideas for how to clarify that it needs to be modified?

Reply:
> Why do you prefer python train.py and do you have any other ideas for how to clarify that it needs to be modified?

I don't strongly prefer it and don't really know what would be less confusing 😅 
I was just thinking that it would be easier for me to rename the file than to update the entire cmd and remove the `<>`

Leaving it up to you.

Reply:
Do you think the brackets:
a. Make it clear this needs to be modified
b. Will be left in by users so that they end up running `<python train.py>`